plugins/backstage-plugin-prometheus/src/components/PrometheusAlertStatus/PrometheusAlertStatus.tsx

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright 2021 Larder Software Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import {
+  InfoCard,
+  Progress,
+  StatusAborted,
+  StatusError,
+  StatusOK,
+  StatusPending,
+  Table,
+  TableColumn,
+} from '@backstage/core-components';
+import { Chip, Tooltip } from '@material-ui/core';
+import Alert from '@material-ui/lab/Alert';
+import { DateTime } from 'luxon';
+import { useAlerts } from '../../hooks/usePrometheus';
+import { PrometheusDisplayableAlert } from '../../types';
+
+export default {
+  title: 'Data Display/Status',
+
+  component: StatusOK,
+};
+
+const state = (str: string) => {
+  switch (str) {
+    case 'firing':
+      return <StatusError>{str}</StatusError>;
+    case 'pending':
+      return <StatusPending>{str}</StatusPending>;
+    case 'inactive':
+      return <StatusAborted>{str}</StatusAborted>;
+    default:
+      return <StatusAborted>Unknown</StatusAborted>;
```

Comment:
would be nice to include `str` in this case so we have a breadcrumb of what the unknown cases are